src/nvim/log.h

```diff
@@ -17,10 +17,11 @@
 #endif
 
 
-#define DEBUG_LOG_LEVEL 0
-#define INFO_LOG_LEVEL 1
-#define WARN_LOG_LEVEL 2
-#define ERROR_LOG_LEVEL 3
+#define TRACE_LOG_LEVEL 0
+#define DEBUG_LOG_LEVEL 1
+#define INFO_LOG_LEVEL 2
+#define WARN_LOG_LEVEL 3
+#define ERROR_LOG_LEVEL 4
```

Comment:
This changes behavior, where `CMAKE_EXTRA_FLAGS += -DMIN_LOG_LEVEL=1` now enables debug messages, whereas it was `INFO` only before..!  (e.g. via https://github.com/neovim/neovim/blob/4174244d8b6718c703f8b8d3e907809bde9430bd/src/nvim/channel.c#L185)

Since it is released in 0.5 already it should probably stay like this, but it seems like documentation (and CI config) needs to be adjusted also (e.g. https://github.com/neovim/neovim/blob/4174244d8b6718c703f8b8d3e907809bde9430bd/.github/workflows/env.sh#L30).